src/_store/AppConfigProvider.js

```diff
@@ -23,9 +24,11 @@ export default class AppConfigProvider extends Component {
         language: PropTypes.string.isRequired,
         theme: PropTypes.string.isRequired,
     };
-
+    
     render() {
         const { children, language, theme } = this.props;
+        addLocaleData(localeData(locale(language)));
```

Comment:
Is aware of that. Wanted to put it in one of the lifeCycle events. Then i noticed it we are not loading on language change , we are just re-rendering based on on the choice and hence the lifecycle properties are not available on the language change . Our locale data on the other hand need to be loaded on each language change and according to the respective selection. :wink: